extensions/elasticsearch-rest-client-common/deployment/src/main/java/io/quarkus/elasticsearch/restclient/common/deployment/DevServicesElasticsearchProcessor.java

```diff
@@ -238,7 +238,7 @@ private GenericContainer<?> createElasticsearchContainer(ElasticsearchDevService
         // Disable disk-based shard allocation thresholds:
         // in a single-node setup they just don't make sense,
         // and lead to problems on large disks with little space left.
-        // See https://www.elastic.co/guide/en/elasticsearch/reference/8.13/modules-cluster.html#disk-based-shard-allocation
+        // See https://www.elastic.co/guide/en/elasticsearch/reference/8.14/modules-cluster.html#disk-based-shard-allocation
```

Comment:
Right... in these comments that probably makes sense.

In documentation though, we need to point to the right version, otherwise the documentation of Quarkus 3.12 in one year will point to a much too recent version.

Reply:
Would you look at that, we don't have link to ES documentation in our own documentation...
Ok, I'll apply your suggestion everywhere then. Thanks!